lib/iris/fileformats/pp_save_rules.py

```diff
@@ -374,7 +374,12 @@ def _grid_and_pole_rules(cube, pp):
         The PP field with updated metadata.
 
     """
```

Comment:
I'd need to double check what happens when saving a non-longitude cross-section. Does this change impact that?!?